utils/clients/admin.ts

```diff
@@ -0,0 +1,16 @@
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
```

Comment:
can you move this dir outside utils ? (top level clients/admin.ts)

Reply:
Fixed in [358e452](https://github.com/pagopa/io-functions-services/pull/68/commits/358e4526d63a6fa870cee31c280a912144c3b5ba)